src/elements/content-sidebar/activity-feed/comment/__tests__/CommentDeleteConfirmation-test.js

```diff
@@ -42,19 +42,5 @@ describe('elements/content-sidebar/ActivityFeed/comment/CommentDeleteConfirmatio
             expect(onDeleteCancelMock).toBeCalled();
             expect(onDeleteConfirmMock).not.toBeCalled();
         });
-
-        test('should handle Enter key', () => {
```

Comment:
this was a bug - it autofocuses the cancel button so enter should just trigger the default handler

Reply:
Had to confirm for myself since I think i added this. `<enter>` on a non-submit cancel button should do the `onclick` (but `<enter>` in a field in a form should do the `onsubmit` of the form.) This was just doing the action of the delete button

https://jsfiddle.net/rs17ep2z/